hoodie-client/src/main/java/com/uber/hoodie/metrics/HoodieMetrics.java

```diff
@@ -79,25 +80,43 @@ private Timer createTimer(String name) {
 
     public void updateCommitMetrics(long commitEpochTimeInMs, long durationInMs, HoodieCommitMetadata metadata) {
         if (config.isMetricsOn()) {
+            String actionType = HoodieActiveTimeline.COMMIT_ACTION;
```

Comment:
why cant we add a new `updateCompactionMetrics`

Reply:
we need to refactor this more nicely for sure, without causing the instanceof checkk. (have not thought very deeply though)